dojo/tools/api_sonarqube/importer.py

```diff
@@ -260,7 +277,11 @@ def import_hotspots(self, test):
                     )
                 )
                 cwe = self.clean_cwe(rule.get("fixRecommendations", ""))
-                references = self.get_references(
+                try:
+                    sonarqube_permalink = f"[Hotspot permalink]({sonarUrl}security_hotspots?id={hotspot['project']}&hotspots={hotspot['key']}) \n"
+                except:
```

Comment:
As higher

Reply:
@kiblik no, these two sonarqube_permalinks are not the same

Reply:
"As higher" was for `except`. URL looks good for me.

Reply:
cp above